packages/esm-patient-registration-app/src/widgets/display-photo.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DisplayPatientPhoto from './display-photo.component';
+
+jest.mock('../patient-registration/patient-registration.resource', () => ({
+  usePatientPhoto: jest.fn().mockReturnValue({ data: { imageSrc: 'test-image-src' } }),
+}));
+jest.mock('geopattern', () => ({
+  generate: jest.fn().mockReturnValue({
+    toDataUri: jest.fn().mockReturnValue('test-pattern-url'),
+  }),
+}));
+
+describe('DisplayPatientPhoto Component', () => {
+  it('should render the component with the patient photo and size should not be small', () => {
+    const patientUuid = '12345';
+    const patientName = 'John Doe';
```

Comment:
We can have these variables globally, right? @ayush-AI So we can use it for the other tests as well.

Reply:
Or you can get it from a mock patient defined [here](https://github.com/openmrs/openmrs-esm-patient-management/blob/main/__mocks__/patient.mock.ts)